content/docs/guides/adopting/from_azure.md

```diff
@@ -598,7 +598,7 @@ From here, you can change the template body and/or surrounding code, rerun `pulu
 
 Now let's see how to actually migrate your ARM-managed resources fully to Pulumi. This requires rewriting the ARM template JSON as real code, either entirely, or one resource at a time. Because you can query deployment outputs and provide parameters in code, you can more easily intermingle ARM-managed resources alongside Pulumi ones. Cyclic dependencies, of course, cannot be expressed, since the entire ARM deployment is seen as one opaque resource to Pulumi.
 
-> Because Pulumi's Azure resource model doesn't match ARM's resource projections exactly, there is no tool currently available to automate this translation. A good apraoch is to copy the ARM template definition into your code and then rewrite it to your language of choice, translating resource and property names as appropriate.
+> Because Pulumi's Azure resource model doesn't match ARM's resource projections exactly, there is no tool currently available to automate this translation. A good approach is to copy the ARM template definition into your code and then rewrite it to your language of choice, translating resource and property names as appropriate.
```

Comment:
```suggestion
> Because Pulumi's Azure resource model doesn't match ARM's resource projections exactly, there is no tool currently available to automate this translation. A good approach is to copy the ARM Template definition into your code and then rewrite it to your language of choice, translating resource and property names as appropriate.
```

Reply:
MS seems to doc as "ARM Template" ... i happened to see this one example ... there are more in the file (and in our docs) ... your choice on whether to do a pass here

Reply:
Microsoft's own documentation doesn't capitalize "Template" so I'm not sure we want to do this?

Reply:
FWIW, the [intro topic](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/overview) in the azure docs keeps template lowercase, so I suggest we follow their lead here. 

At Microsoft, casing for products and features usually go through a branding pass with marketing folks, so if it's lowercase in their docs, it should probably be lowercase in our docs.